backend/ltsa/Configuration/LtsaOptions.cs

```diff
@@ -0,0 +1,59 @@
+namespace Pims.Ltsa.Configuration
+{
+    /// <summary>
+    /// LtsaOptions class, provides a way to configure the LtsaService.
+    /// </summary>
+    public class LtsaOptions
+    {
+        #region Properties
+        /// <summary>
+        /// get/set - The URI to the Ltsa API service.
+        /// </summary>
+        public string HostUri { get; } = "https://tdtest3-x42b.ltsa.ca/titledirect/search/api";
```

Comment:
Seems like the default is the same as what you have in the configurations.  This one has a path included.